docs/dependencies.md

```
@@ -0,0 +1,163 @@
+# Dependencies
+
+## Core Spring Boot Dependencies
```

Markdownlint (0.37.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)